src/api.ts

```diff
@@ -176,6 +178,8 @@ export function parseJsonConvertingSnakeToCamel(str: string): AuthenticationInfo
             this.legacyUserId = value
         } else if (key === "impersonator_user") {
             this.impersonatorUserId = value
+        } else if (key === "login_method") {
+            this.loginMethod = value
```

Comment:
@itailevi98 , this is where I added the snake- to camelcase conversion for loginMethod. I leave the values as snake case though.

Reply:
Right, this is what I mean -- the `LoginMethod` type is camelCase, but the value will be snake_case, meaning there's an inconsistency here -- we should update this value here to also be camelCase. This is what we do in [the node library](https://github.com/PropelAuth/node/pull/32/files#diff-b380de27e83ad3ba731f62c4887b9de8e28c840804c9a07bff7496739db64471R70)

Reply:
Copy that. But I think the above logic still recursively covers that since the only keys in the body of the LoginMethod type that need converting are "login_method" and "org_id" which are both covered in `parseJsonConvertingSnakeToCamel`. It's also nice to have the one util that handles all snake2camel conversion rather than spreading that requirement out.